ConnectSecure/Deploy-ConnectSecure-V4.ps1

```diff
@@ -289,7 +296,7 @@ if(!$CompanyID) {
 # if(!$Uninstall) {
 Write-Host "Company ID from $customCompanyID Custom Doc Field or passed to script: $CompanyID"
 
```

Comment:
Should probably add a $userSecret validation test for at least length or character types somewhere in here as well since the others are validated. But probably fine for first pass to skip it. Maybe even just a length test for some minimum length with an error if none is included since installation will fail, so there's no point in proceeding with the download if it's blank.

Reply:
Added some basic validation to check that it isn't blank. It looks like there might be some structure to the token, so we could probably improve this.